internal/tools/go.mod

```diff
@@ -1,20 +1,21 @@
 module github.com/uber/cadence/internal/tools
```

Comment:
hmmm.  something in here is breaking mockery, it produces an empty s3iface mock 🤔 

might have to finish that v3 migration before any of this will work.